app/helpers/reactjs_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ReactjsHelper
+
+  def mount_react_component(name, selector, data)
```

Comment:
I understand how it works, I am suggesting to get rid of the boilerplate `<div id="statistics"></div>` and just create it inside the helper, using a randomized ID to make sure we don't get conflicts.